src/runtime/virtcontainers/clh.go

```diff
@@ -170,6 +170,7 @@ type cloudHypervisor struct {
 	APIClient      clhClient
 	ctx            context.Context
 	id             string
```

Comment:
@fidencio This is more of a comment for the commit message
"Although it looks like it doesn't make any difference of what we're
doing of what we used to do," 

It isnt really clear to me what you are trying to convey here.